src/objects/zcl_abapgit_objects_files.clas.abap

```diff
@@ -170,8 +170,10 @@ CLASS ZCL_ABAPGIT_OBJECTS_FILES IMPLEMENTATION.
 
   METHOD add_xml.
 
-    DATA: lv_xml  TYPE string,
-          ls_file TYPE zif_abapgit_definitions=>ty_file.
+    DATA: lv_xml        TYPE string,
+          lv_hex(1)     TYPE x VALUE '23',
```

Comment:
suggest

```suggestion
          lv_hex     TYPE x LENGTH 1 VALUE '23',
```

similar for the next definition

Reply:
ok. I will change